picard/ui/options/interface.py

```diff
@@ -36,7 +37,35 @@ class InterfaceOptionsPage(OptionsPage):
     PARENT = "advanced"
     SORT_ORDER = 40
     ACTIVE = True
-
+    SEPERATOR = u'—'*5
+    TOOLBAR_BUTTONS = {'add_directory_action': N_(u'Add Folder'),
+                       'add_files_action': N_(u'Add Files'),
+                       'cluster_action': N_(u'Cluster'),
+                       'autotag_action': N_(u'Lookup'),
+                       'analyze_action': N_(u'Scan'),
+                       'browser_lookup_action': N_(u'Lookup in Browser'),
+                       'save_action': N_(u'Save'),
+                       'view_info_action': N_(u'Info'),
+                       'remove_action': N_(u'Remove'),
+                       'submit_action': N_(u'Submit AcoustIDs'),
+                       'play_file_action': N_(u'Open in Player'),
+                       'cd_lookup_action': N_(u'Lookup CD...'),
+                       }
+    TOOLBAR_ICONS = {'add_directory_action': 'folder',
+                     'add_files_action': 'document-open',
+                     'cluster_action': 'picard-cluster',
+                     'autotag_action': 'picard-auto-tag',
+                     'analyze_action': 'picard-analyze',
+                     'browser_lookup_action': 'lookup-musicbrainz',
+                     'save_action': 'document-save',
+                     'view_info_action': 'picard-edit-tags',
+                     'remove_action': 'list-remove',
+                     'submit_action': 'acoustid-fingerprinter',
+                     'play_file_action': 'play-music',
+                     'cd_lookup_action': 'media-optical',
+                     }
```

Comment:
Is there a reason to duplicate keys here instead of using dict/subdict:

```python
TOOLBAR =  { ''add_directory_action': {label: N_(u'Add Folder'), icon: 'folder'}, .... }
```